upload/lambdas/health_check/health_check.py

```diff
@@ -0,0 +1,109 @@
+from datetime import datetime, timedelta
+import json
+import os
+
+import boto3
+import requests
+
+from upload.common.database import _run_query
+from upload.common.logging import get_logger
+from upload.common.upload_config import UploadConfig
+
+logger = get_logger(__name__)
+
+client = boto3.client('cloudwatch')
+
+
+class HealthCheck:
+    def __init__(self):
+        self.env = os.environ['DEPLOYMENT_STAGE']
+        logger.debug(f"Running a health check for {self.env}. Results will be posted in #upload-service")
```

Comment:
Why is this debug vs info?

Reply:
following the convention in the csum lambda, I can switch it to info